packages/tentales/src/middleware/middlewares/protect-service-routes-middelware.ts

```diff
@@ -1,21 +1,26 @@
 import { Middleware } from "tentales"
+import { path } from "ramda"
 import { isServiceRoute } from "../../utils/url"
 import { readToken } from "../../utils/token"
 import { verify } from "jsonwebtoken"
 import { AuthError } from "../errors"
 
 export const protectServiceRoutesMiddleware: Middleware = ({ config }) => {
   return async function actualProtectServiceRoutesMiddleware(ctx, next) {
-    if (!isServiceRoute(ctx.request.URL.pathname)) {
-      await next()
-    } else {
+    if (
+      isServiceRoute(ctx.request.URL.pathname) &&
+      path(["auth", "serverSecret"], config)
```

Comment:
This shouldn't be needed as we're only using the middleware if server secret is set.

Reply:
Well, since `auth` and `serverSecret` are marked as optional typescript will report an error if I just do `config.auth.serverSecret`, even though we know this won't be an issue.